drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-api/src/main/java/org/drools/workbench/screens/scenariosimulation/model/Simulation.java

```diff
@@ -109,6 +109,24 @@ public void resetErrors() {
         scenarios.forEach(Scenario::resetErrors);
     }
 
+    public Simulation cloneSimulation() {
+        Simulation toReturn = new Simulation();
+        // TODO CHECK ALL THE FOLLOWING *SHOULD BE* REAL CLONES
```

Comment:
Would be better to make tickets for the TODO's. These tend to be left into the code and never checked.